### ISSUE TITLE
Weird behavior when planning roads with infinite radius

### ISSUE THREAD

**Issue by kingoflolz** [citybound/citybound#121]
To be perfectly honest, I was not sure what I was expecting, but if you try to build a circle with almost zero curvature or infinite radius, weird stuff happens.

![rsz_screenshot_from_2017-02-02_18-58-00](https://cloud.githubusercontent.com/assets/8016247/22541481/82ac505c-e97a-11e6-83cf-5e5790147cde.png)

**Comment by aeickhoff** [comment]
Yeah, that's just floating point inaccuracy. Also filing it as a UI issue to maybe prevent that from happening at all
